src/halmos/__main__.py

```diff
@@ -949,7 +949,7 @@ def _solve_end_to_end_callback(
             print(ctx.traces[path_id], end="")
 
         if model.is_valid:
-            print(color_error(f"Counterexample: {model}"))
+            print(color_error(f"Counterexample: [{path_id}] {model}"))
```

Comment:
this prints as `Counterexample: [N]`, but it might not be obvious to users what the number N means. could you make it to explicitly say that N is the path id?